client/service_logs_test.go

```diff
@@ -71,21 +70,27 @@ func TestServiceLogs(t *testing.T) {
 		},
 		{
 			options: types.ContainerLogsOptions{
-				// An complete invalid date, timestamp or go duration will be
-				// passed as is
-				Since: "invalid but valid",
+				// timestamp will be passed as is
+				Since: "1136073600.000000001",
 			},
 			expectedQueryParams: map[string]string{
 				"tail":  "",
-				"since": "invalid but valid",
+				"since": "1136073600.000000001",
 			},
 		},
+		{
+			options: types.ContainerLogsOptions{
```

Comment:
Synced the `client/service_logs_...` files with `client/container_logs_..` ones.

Looking at this type, I wonder if we should create  a `types.ServiceLogsOptions`, because service-logs and container-logs are now different in functionality (can possibly addressed, but there's no guarantee that they always have feature parity)

@dnephin wdyt?